quickwit-indexing/src/actors/indexing_server.rs

```diff
@@ -320,16 +320,16 @@ impl IndexingServer {
     ) -> anyhow::Result<IndexingPipelineId> {
         let pipeline_id = IndexingPipelineId {
             index_id,
-            source_id: "void-source".to_string(),
+            source_id: "vec-source".to_string(),
```

Comment:
The void source is better for those tests because, with a vec source, the whole pipeline can finish before we observe it running.